ISISComputingGroup/ibex_gui
# Ticket1015 remove classes after pv refactoring

1024 - https://github.com/ISISComputingGroup/ibex_gui/pull/153 should be reviewed before this, as the changes on that branch are also reflected here.

To test:
*Everything still builds
*Check unit tests still run (via Tycho and Eclipse)
*No obvious problems especially relating to PV switching/displaying
